recipes/nltk-data/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "nltk" %}
+{%set version = "2017.10.22" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "14b55df9314ece2e7e53258c41b2f29a1fe0d7517ded586fa9ed137edf9a39ec" %}
+
+package:
+  name: {{ name }}-data
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/nltk/nltk_data/archive/gh-pages.zip
```

Comment:
This is no good; a rebuild will always get the current version, but tag it as `2017.10.22`.

You could instead do something like, right after the `version` tag at the top:
```html+jinja
{% set commit = "c0a373066e1477bd201acb071be171e8cc0ca673" %}
```
and then:
```yaml
  url: https://github.com/nltk/nltk_data/archive/{{ commit }}.tar.gz
```

Reply:
Done. I thought the archive would give me master every time but this is better.